jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
+    "strictNullChecks": true
+  }
+}
```

Comment:
Does that mean that the projects importing `github-project` will need the setting as well?

Reply:
"Need" only in the sense that if they want to see `null` types they either need strict mode or `strictNullChecks` enabled. It works as expected without that setting enabled at the TS level, so `string | null` just looks like `string`